app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<div>
+  <h1><%= @user.name.capitalize %>'s Events</h1>
+  <i class="fa fa-calender" aria-hidden="true"></i></span>
+  <% if @events_by_current_user %>
+    <h3>Created Events</h3>
+    <%= render @events_by_current_user %>
+
+    <div>
+    <i class="fa fa-calender" aria-hidden="true"></i></span>
+    <h3>Upcoming Events</h3>
+    <%= render @upcoming_events %>
+    </div>  
+    <div>
+    <i class="fa fa-calender" aria-hidden="true"></i></span>
+    <h3>Previous Events</h3>
+    <%= render @prev_events %>
```

Comment:
render methods in a view file expects a file path. Go through: 3.4, Using Partials  on
 
```https://guides.rubyonrails.org/layouts_and_rendering.html```